app/models/user.rb

```diff
@@ -53,7 +53,7 @@ class User < ApplicationRecord
 
   validates :full_name, presence: true
   validates :role, presence: true, if: -> { email_authentication.present? }
-
+  # validates :access_level, presence: true, if: -> { email_authentication.present? }
```

Comment:
Keeping this off during the migration I guess? If we haven't already, let's add a story to turn it back on so we don't forget.

Reply:
Yup, I added a comment in the [tests](https://github.com/simpledotorg/simple-server/pull/1185/files#r470619885) for this, forgot to add the comment here. I'll add a story as well.